package.json

```diff
@@ -60,7 +62,8 @@
     "percy": "percy exec -- jest --config jest.visual.config.js --runInBand",
     "visual-testing-app:build": "NODE_ENV=production webpack --config visual-testing-app/webpack.config.js",
     "visual-testing-app:build:watch": "NODE_ENV=production webpack --config visual-testing-app/webpack.config.js --watch",
-    "visual-testing-app:serve": "serve visual-testing-app/dist --no-clipboard --single --listen 3001"
+    "visual-testing-app:serve": "serve visual-testing-app/dist --no-clipboard --single --listen 3001",
+    "typecheck": "dtslint typings"
```

Comment:
I wouldn't call the script "typecheck" since we don't have any types to check...

Reply:
I don't understand your point, we do have types. 🤔

Reply:
dom-testing-library calls it dtslint, which is a better name.

https://github.com/testing-library/dom-testing-library/blob/4b2e8a535d8e88bc03126307cbee48a74dbf547e/package.json

We aren't using typescript, so calling a script "typecheck" feels weird. Does this script test our PropTypes? No...

Reply:
From the https://github.com/microsoft/dtslint readme:

> `dtslint` tests a TypeScript declaration file for style and correctness. It will install `typescript` and `tslint` for you, so this is the only tool you need to test a type definition.

So yes, it obviously uses typescript to be able to perform type checks.

> dom-testing-library calls it dtslint

I know, but in all our repos when we use types we call this script `typecheck`, so I just went with that.

I think you're misunderstanding a bit what this tool does...

Reply:
If you create a pull request with one line of text for it's summary, it's not unusual for people to misunderstand it...

Reply:
Sure but to be fair it's a "draft" PR, I haven't asked for a review yet and it's marked as experimental 🙂